.github/workflows/ci_coverage.yml

```diff
@@ -0,0 +1,139 @@
+name: SeqAn3 CI on Linux
+
+on:
+  push:
+    branches:
+      # Push events to branches matching refs/heads/master
+      - 'master'
+      # Push events to branches matching refs/heads/release*
+      - 'release*'
+  # Trigger after PR was unlabeled
+  pull_request:
+    types:
+      - unlabeled
+  # Enables a manual trigger, may run on any branch
+  workflow_dispatch:
+
+# Do not cancel on push events
+concurrency:
+  group: coverage-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: false
```

Comment:
is this in general or only when merging on master (I'm still not good in reading github action syntax)

Reply:
Syntax says: always.
But the clang-format (which only runs on PRs) CI will always cancel all other CI.
So, in practice, it is just never canceled on Master.

Reply:
can you drop a note about this here? Since it is not obvious and future enricos might wonder why it is cancelled in PRs